docker-compose/docker-compose.yml

```
@@ -212,6 +212,38 @@ services:
       --server_id            ${SERVER_ID:-myserver}
       --core_host            ${CORE_HOSTS:-core}
       --enable_ops_metrics   true
+  monitoring_edge:
+    image: ns1inc/privatedns_monitoring_edge:${TAG:-2.5.1}
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+    environment:
+      CONFIG_PORT: 3308
+      CONTAINER_NAME: ${MONITORING_EDGE_CONTAINER_NAME:-monitoring_edge}
+    restart: unless-stopped
+    ports:
+      - "3308:3300" # http configuration
+    healthcheck:
+      test: supd health --check
+      interval: 15s
+      timeout: 10s
+      retries: 3
+    volumes:
+      - type: volume
+        source: ns1monitoringedge
+        target: /ns1/data
+        volume:
+          nocopy: true
+    command: >-
+      --pop_id                 ${POP_ID:-mypop}
+      --server_id              ${SERVER_ID:-myserver}
+      --core_host              ${CORE_HOSTS:-dist}
+      --enable_ops_metrics     true
+      --digest_service_def_id  4
```

```suggestion
      --digest_service_def_id  3
```
Using bootstrap will add two service defs `dns` (1) + `dhcp` (2). If the third we add is monitoring, then that would be (3).